grammar/haxe.bnf

```diff
@@ -490,8 +496,16 @@ left shiftExpression ::= shiftOperator additiveExpressionWrapper
 private additiveExpressionWrapper ::= multiplicativeExpressionWrapper additiveExpression*
 left additiveExpression ::= ('+' | '-') multiplicativeExpressionWrapper
 
-private multiplicativeExpressionWrapper ::= prefixExpression multiplicativeExpression*
-left multiplicativeExpression ::= ('*' | '/' | '%') (prefixExpression | suffixExpressionWrapper)
+private multiplicativeExpressionWrapper ::= assignExpressionWrapper multiplicativeExpression*
+left multiplicativeExpression ::= ('*' | '/' | '%') assignExpressionWrapper
+
+/* Technically, assignExpressions don't have higher priority than everything else.
+ * But for parsing, this makes sense; otherwise, assignment operators that comprise
```

Comment:
Does this really work? Assignment expression are supposed to  have the *lowest* precedence in order to parse `a = b + c` correctly. If you give them higher priority, you would end up with `(a = b) + c` instead. How do you prevent that (or does it simply not matter in this context)?